README.md

```diff
@@ -1,26 +1,25 @@
-# Polyphona
+# polyphona
```

Comment:
Seems like this has erased the README we already had. Can you please merge them together?

Reply:
Done.